### PR TITLE
[DRAFT] patch release 0.8.1

### PR DESCRIPTION
**Pull request by felixdittrich92**
This PR:

- pypi images fix (hopefully)
- broken pypi changelog url fix
- version 0.8.1

### CODE REVIEW

**Comment by felixdittrich92** [comment]
@odulcy-mindee 
With the images it should work now tested on https://dillinger.io/

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -108,4 +108,6 @@ jobs:
         shell: bash -el {0}
         run: |
           conda config --set channel_priority strict
+          conda install pytorch torchvision torchaudio cpuonly -c pytorch
           conda install -c techMindee -c pypdfium2-team -c bblanchon -c conda-forge python-doctr
+          python -c "import doctr; print(doctr.__version__)"
```
Comment (odulcy-mindee) [inline .github/workflows/publish.yml:113]: Did you check locally it worked ?

File: README.md
Comment (odulcy-mindee) [inline README.md:1]: Can you edit the link in "[We offers Docker container support for easy testing and deployment](https://github.com/mindee/doctr/packages)." to be https://github.com/mindee/doctr/pkgs/container/doctr please ?
Also, there is a typo (We offer without S)

Comment (odulcy-mindee) [inline README.md:1]: Also, links to LICENSE and CONTRIBUTING are broken on pypi
image

**Review by odulcy-mindee — CHANGES_REQUESTED** [review]

File: README.md
Comment (felixdittrich92) [reply README.md:1]: Sure :)

Comment (felixdittrich92) [reply README.md:1]: Ah good catch will fix it 👍

**Comment by felixdittrich92** [comment]
@odulcy-mindee Mh there is still a problem with the conda package normally the import test should work i will test a few things before

**Review by odulcy-mindee — DISMISSED** [review]
Awesome, thanks !

**Comment by felixdittrich92** [comment]
#1493

**Pull request closed without merging**